dev/breeze/src/airflow_breeze/utils/cdxgen.py

```diff
@@ -280,9 +281,6 @@ def build_all_airflow_versions_base_image(
     main new commit. Tags used are:
 
     main_ci_images_fixed_tags = {
-        "3.6": "latest",
```

Comment:
We should leave those in - we use it to generate SBOMs for old versions of Airlfow and we need all historical versions of Python in this mapping.

Reply:
I will restore but... are you sure? This is in pydoc... not in active code :-D

Reply:
Ah right 🤯

Reply:
You can remove it - this part is anyhow something that we will have to improve a bit later  - when we improve SBOM generation (there are few things to do in the future).